types/hapi__jwt/index.d.ts

```diff
@@ -0,0 +1,442 @@
+// Type definitions for @hapi/jwt 2.0
+// Project: https://github.com/hapijs/jwt
+// Definitions by: Sergio Sánchez <https://github.com/ssanchezmarc>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 4.2
```

Comment:
Why was this required?

Reply:
I try to follow the same conventions I saw in other similar cases where we have defined types for hapi modules. No sure if it is mandatory or just optional :)